lib/fuelsdk/client.rb

```diff
@@ -48,6 +48,7 @@ class Client
 		def jwt= encoded_jwt
 			raise 'Require app signature to decode JWT' unless self.signature
 			decoded_jwt = JWT.decode(encoded_jwt, self.signature, true)
+			decoded_jwt = decoded_jwt[0] if decoded_jwt.is_a?(Array)
```

Comment:
```suggestion
			decoded_jwt = JWT.decode(encoded_jwt, self.signature, true).is_a?(Array) ? JWT.decode(encoded_jwt, self.signature, true) : JWT.decode(encoded_jwt, self.signature, true)[0]
```

Reply:
this way we're going to run decode twice. Also, it's a really long ternary operator that is not as readable.
To make it oneline I have 2 proposals (same mechanism different syntax):
```ruby
#1
[JWT.decode(encoded_jwt, self.signature, true)].flatten
#2 
Array.new.push(JWT.decode(encoded_jwt, self.signature, true)).flatten
```

Reply:
430d5e4